frontend/components/home/Banner.tsx

```
@@ -0,0 +1,155 @@
+                            <h3 className='title' data-aos='fade-right' data-aos-delay='400'>
+                                Never Stop{''}
+                                <span className='relative ml-3'>
+                                    <span className='svg-icon'>
+                                        <svg
+                                            width='100%'
+                                            height='100%'
+                                            viewBox='0 0 61 68'
+                                            fill='none'
+                                            xmlns='http://www.w3.org/2000/svg'
+                                            aria-label='learning-curve'
+                                        >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)